internal/pkg/node/node.go

```diff
@@ -180,6 +183,17 @@ func (node *Node) join(remote scale.RemoteNode) {
 		node.sugar.Fatal("no successor to predecessor found")
 	}
 
+	// ok this is what the new function call would look like
```

Comment:
https://github.com/a8m/reflect-examples#call-to-a-function-dynamically-similar-to-the-templatetext-package